docs: suggestions for updates to cluster docs (#4253)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.2/clusters/applications/index.md

## Application Dashboards

Applications, formerly called Platform Services and Addons, are installed by the management cluster. You can visit a cluster's detail page to see the application dashboards enabled from the deployed applications under the **Application Dashboards** section.

## Applications

Under the **Applications** section of the cluster's detail page, you can view the workspace applications enabled for the cluster, grouped by category.

![Cluster Detail Page - Applications](../../img/cluster-detail-page-applications.png)

> In this section, you can also view the current status of the enabled applications on the cluster. The status is displayed on the application card. Hovering on the status displays details about the status of the application.

Reviewer: ```suggestion
In this section, you can also view the current status of the enabled applications on the cluster, on each application card. Hovering on the status displays details about the status of the application.
```
